document/js-api/index.bs

```diff
@@ -1101,12 +1100,12 @@ Note: Exported Functions do not have a \[[Construct]] method and thus it is not
         1. [=Clean up after running script=] with |relevant settings|.
         1. Assert: |result|.\[[Type]] is <emu-const>throw</emu-const> or <emu-const>normal</emu-const>.
         1. If |result|.\[[Type]] is <emu-const>throw</emu-const>, then:
-            1. If |v| [=implements=] {{Exception}},
-                1. Let |type| be |v|.\[[Type]].
-                1. Let |payload| be |v|.\[[Payload]].
+            1. If |result| [=implements=] {{Exception}},
```

Comment:
*result* is a completion record, so can't be an object. I think a "Let *v* be *result*.[[Value]]" is missing.

Reply:
Isn't that just `Let _v_ be _result_`? Below we are using `v.[[Type]]` and `v.[[Payload]]`. I don't think `result.[[Value]].[[Type]]` makes sense..?

Reply:
I thought this meant `[[Type]]` and `[[Payload]]` exist within [the completion record](https://tc39.es/ecma262/#sec-completion-record-specification-type), because it looks this line says so: https://github.com/WebAssembly/exception-handling/blob/ca2b7155837c136ce8b3c95c7e49afdd60974016/document/js-api/index.bs#L1043
It looks the completion record is extended to contain a tag and a payload. 

But as you said, `WebAssembly.Exception` also has `[[Type]]` and `[[Payload]]` field. Do these two fields exist within both entities, `WebAssembly.Exception` and the completion record?

Reply:
I don't follow. `func_invoke` is a wasm algorithm, so returns wasm values, not completion records.

Reply:
OK, I reverted this too.